src/tigerbeetle/cli.zig

```diff
@@ -756,7 +756,7 @@ fn parse_args_start(start: CLIArgs.Start) Command.Start {
         "development", "experimental",
     };
     inline for (std.meta.fields(@TypeOf(start))) |field| {
-        @setEvalBranchQuota(2_200);
+        @setEvalBranchQuota(3_000);
```

Comment:
This one I agree with! I'd even bump to 4k: when you increment something, make it at least a (binary) order of magnitude larger!